### PR TITLE
Bootstrap 5

### LINKED ISSUE #683: Bootstrap 5
I had a chat with @MyIgel about upgrading to Bootstrap 4. I did some investigation and here are some issues I found that we should think about:

- The current engelsystem is implemented using Less. A proper migration should then also cover switching from Less to Sass.
- Date datepicker doesn't seem to receive any updates. So I guess this has to be changed as well.
- This is a quite huge change. I would start that after the Congress

Any opinions about these points?

### PR DESCRIPTION
**Pull request by weeman1337**
Migrates the Engelsystem to Bootstrap 5 

closes #683

### CODE REVIEW

File: config/config.default.php [hunk 1]
```diff
@@ -126,6 +126,14 @@
         '1'  => 'Engelsystem dark',
     ],
 
+    'theme_info' => [
+        '1' => [
+            'navbar'        => 'dark',
```
[the remark this answers is not in the document]
Comment (MyIgel) [reply config/config.default.php:131]: I would have just searched for "dark" in the name, but that's also an option :D

**Comment by MyIgel** [comment]
After the general markup changes are done we just need to fix the funny part of the design ;)

There are some small steps to do:

* check for theme variables that got removed / are not used anymore
* Resolve the `base.scss @todo`
* Fix the styling of all designs, especially on theme 7/35c3
  * button colors
  * headings color
  * forms bg

And all the other design glitches and quirks that ocured :upside_down_face:

**Comment by MyIgel** [comment]
This PR is finally ready :heart_eyes: -> @msquare plz review ^^

As its already merged pushing it to the main branch should be enough ;)

**Review by msquare — CHANGES_REQUESTED** [review]
* Scrolling table headers in shift view have a transparent background
* Maintenance page css include is broken
* Login: Input-group-text background does not fit theme/high contrast

File: includes/sys_form.php [hunk 2]
```diff
@@ -72,7 +70,7 @@ function form_date($name, $label, $value, $start_date = '', $end_date = '')
     return form_element($label, '
     <div class="input-group date" id="' . $dom_id . '" data-min-date="' . $start_date . '" data-max-date="' . $end_date . '">
```
[the remark this answers is not in the document]
Comment (weeman1337) [reply includes/sys_form.php:71]: > Maintenance page css include is broken

✓

**Comment by weeman1337** [comment]
- [ ] Scrolling table headers in shift view have a transparent background
- [x] Maintenance page css include is broken
- [ ] Login: Input-group-text background does not fit theme/high contrast

**Comment by weeman1337** [comment]
* Date/time pickers added to the design page
* Date/time values from input are now taken over to the picker and vice versa

**Comment by msquare** [comment]
Great work everybody! Thank you so much for upgrading Bootstrap and all the existing themes.

**Pull request merged**